### PR TITLE
aded info about google_cloud json data and how to obtain that.

### PR DESCRIPTION
**Pull request by andlo**
Removed the info about google cloud TTS as I understand from Åke, that this wassnt right. Only module google is working.

### CODE REVIEW

File: _pages/languages.md [hunk 1]
```diff
@@ -92,27 +136,22 @@ TTS engines are made available by different vendors, and they each have differen
...
   “tts”: {
-  “module”: “google_cloud”,    <- this should be set to the name of your TTS provider (ie "google_cloud", "wit" etc)
-  “google_cloud”: {            <- this should be set to the name of your TTS provider (ie "google_cloud", "wit" etc)
-    “lang”: “hi-in”,           <- the IETF BCP-47 language code for your language (shown is Hindi as spoken in India)
-    “credential”: {            <- some STT engines require credentials - check the documentation for the STT engine
-      “json”: {
-      }
-    }
+  “module”: “google”,
+  “lang": "hi"
```
Comment (forslund) [inline _pages/languages.md]: This looks incorrect: I think It should be

```json
  "tts": {
    "module": "google",
    "google": {
      "lang": "hi"
    }
  }
```

And maybe there could be an indication that there are several options other than google?

Comment (andlo) [reply _pages/languages.md]: yah that looks right

**Comment by andlo** [comment]
What do I do.....Do I make the changes from you manual and make a new PR or is there some other clever way ?

**Comment by forslund** [comment]
Thank you @andlo. You totally rocks the docs! Merging

**Pull request merged**